### ISSUE TITLE
Custom border on fixed row

### ISSUE THREAD

**Issue by nrmohan** [handsontable/handsontable#4142]
Bottom border on fixed row is overlapping making the border thicker and while horizontal scrolling the border is moving independently. 

### Demo
http://jsfiddle.net/hn7ae6vs/1/

**Comment by AMBudnik** [comment]
I get something like this with the bottom border on Chrome 56 (Win10). Do you get the same result?

![border](https://cloud.githubusercontent.com/assets/10757813/24094066/49c21690-0d57-11e7-9412-745fa32b63d2.gif)

**Comment by nrmohan** [comment]
Yes. see the red border in the bottom. it is moving under the row header. Also the thickness of the border is getting effected.

**Comment by AMBudnik** [comment]
Thanks for confirming @nrmohan

**Comment by AMBudnik** [comment]
The issue is still replicable in v 0.34.3 http://jsfiddle.net/hn7ae6vs/1/

**Comment by AMBudnik** [comment]
The issue is replicable in v `2.0.0.` and doesn't seem to work fine in any of the previous versions.

**Comment by AMBudnik** [comment]
workes the same in v 4.0.0

**Comment by AMBudnik** [comment]
The issue still looks the same in v 5.0.0. The `width` is too big so the table's headers do not cover the border

**Comment by aninde** [comment]
This issue is fixed at currently tested version `7.2.2` with new `svg` customBorders. 

http://jsfiddle.net/aninde/419atbqr/ 7.2.2 still broken
http://jsfiddle.net/aninde/e5g1x9sb/ fixed - 7.2.2 version with new SVG drawing

![4142-svg](https://user-images.githubusercontent.com/30186818/69814903-7e30ab80-11f5-11ea-9b46-22f666e89f3f.gif)

**Comment by aninde** [comment]
This bug is fixed on Mac Os Catalina 10.15:
- Chrome 78
- Safari 13
- Firefox 70

and Windows 10:
- IE 11
- Edge
- Edge Dev

We should add tests to this so that this error does not recur.
